cs/homepage.texy

```diff
@@ -67,26 +67,26 @@ Testování
 
 Utility
 -------
-- [Datum a čas |datetime]
+- [Pole |arrays] - manipulace s poli
+- [Datum a čas |datetime] - ovládání času
```

Comment:
Je to schvalne, aby to nalakalo, stejne jako `protože stahování zipů je tááák zastaralé`